frontend/src/components/experimentRuns/ExperimentRunsListContent.js

```diff
@@ -70,8 +70,11 @@ const getTableColumns = (runTypes, instruments, launchesById) => [
     dataIndex: "run_processing_launch_time",
     sorter: true,
     render: (_, experimentRun) => (
-      <div style={{minWidth: "12rem"}}>
-        <ExperimentRunLaunchCard experimentRun={experimentRun} experimentRunLaunch={launchesById[experimentRun.id]}/>
+      <div style={{ minWidth: "12rem" }}>
+        {
+          runTypes.itemsByID[experimentRun.run_type].name !== "Axiom" &&
```

Comment:
The run type field needs_run_processing will give us directly the information without having to do an exception for Axiom. If we ever add another run_type we'll set needs_run_processing to true if we want the widget and false if we don't.